Project_Xcavate.md

```diff
@@ -0,0 +1,158 @@
+
+- **Project Name:** Xcavate
+- **Team Name:** Xcavate
+- **Payment Address:** 4pjePvJgUZAMKtnSmYJitGacxBRnsKe1GvCb1t6vgg2Kof8o 
+- **Level:** 1 = $10000
+
+
+## Project Overview
+
+This is the initial grant application
+
+### Overview
+
+Xcavate will be a first-of-its-kind real-world land & property NFT that can be collateralized to finance development transactions. 
+Xcavate facilitates digitizing land and/or property title deeds through minting and bundling of NFTs, which are then linked to verifiable digital identities (DIDs) approved by government entities.
+Xcavate is an ecosystem that allows real-world property developers access to finance parcels of physical land with planning permission for development. Once the land is at the final development stage, the developer will create NFTs for the individual properties to be sold/managed by a peer-to-peer marketplace (property owners, investors, trusts etc.). 
+The system will significantly reduce the time from real estate planning permission to build start. By simplifying the finance application process and speeding up the release of funds to the developer.. Xcavate not only provides real-world property developers with a way to build quicker but, as a result, enables a more significant number of much-needed houses to be delivered to market. It also provides liquidity to static real-world assets at every stage of the property development process in a fair and risk-averse way. It will enable the real estate to be onboarded cleanly and efficiently, opening up opportunities throughout the broader Dotsama ecosystem.
+
+![](https://xcavate.io/wp-content/uploads/2022/11/XCAV-WEB-3-Grant-Application-Image-1.jpg)
+
+![](https://xcavate.io/wp-content/uploads/2022/11/XCAV-WEB-3-Grant-Application-Image-2.jpg)
+
+![](https://xcavate.io/wp-content/uploads/2022/11/XCAV-WEB-3-Grant-Application-Image-3.jpg)

```

Comment:
Only two of these images load for me. Interestingly, all the xcavate.io URLs return 404 for me.